io.github.rafostar.Clapper.Enhancers.Default.metainfo.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>io.github.rafostar.Clapper.Enhancers.Default</id>
+  <extends>io.github.rafostar.Clapper.Enhancers</extends>
```

Comment:
It extends the app, extending an extension point makes no sense. `com.github.rafostar.Clapper`

Reply:
OK, renamed `io.github.rafostar.Clapper.Enhancers.Default` -> `io.github.rafostar.Clapper.Enhancers` and made it simply extend Clapper app.